### PR TITLE
Alt body

### PR DESCRIPTION
**Pull request by thecapacity**
Setup an alternate route for parsing page body

### CODE REVIEW

**Comment by thecapacity** [comment]
Still working on this - hold pull for now

**Comment by thecapacity** [comment]
@khandelwal I think this is good to merge now -- it should give the first approximation of functionality

File: scraper/fedtext/pipelines.py [hunk 1]
```diff
@@ -5,7 +5,76 @@
...
+        item['text_list'] = [word for word in item['text_list'] if word not in self.dead_text]
+        return item
+
+class Split(object):
+    ''' Used in pipeline processing to split words '''
+    def process_item(self, item, spider):
+        item['word_list'] = tokenizer.tokenize( "".join( item['text_list'] ) )
+        return item
+
+class Filter(object):
```
Comment (khandelwal) [inline scraper/fedtext/pipelines.py:65]: Do these one method classes actually have to be classes?

File: scraper/fedtext/spiders/tutorial_spider.py [hunk 2]
```diff
@@ -40,4 +40,8 @@ def parse(self, response):
         visible_texts = [t.strip() for t in texts if self.visible(t)]
         item = FedtextItem()
         item['text_list'] = visible_texts
+        item['word_list'] = []
```
Comment (khandelwal) [inline scraper/fedtext/spiders/tutorial_spider.py:43]: Should we be initializing these to be an empty list?

**Comment by thecapacity** [comment]
Also note though - I had to use the NLTK Downloader to get the corpus -- run a python REPL and try it and it'll tell you what to run, something like `nltk.downloader()`

File: scraper/fedtext/spiders/tutorial_spider.py (hunk 2 above)
Comment (khandelwal) [inline scraper/fedtext/spiders/tutorial_spider.py:43]: Can we do this in the constructor?

Comment (thecapacity) [inline scraper/fedtext/spiders/tutorial_spider.py:43]: Ideally maybe - but I couldn't make it work. Maybe you'll have more success.

**Pull request merged**